packages/p2p/src/components/app.jsx

```diff
@@ -54,7 +54,10 @@ class App extends Component {
 
         /* if there is no error means it's an advertiser else it's a client */
         if (!advertiser_info.error) {
-            await this.setState({ advertiser_id: advertiser_info.p2p_advertiser_info.id, is_advertiser: true });
+            await this.setState({
+                advertiser_id: advertiser_info.p2p_advertiser_info.id,
+                is_advertiser: !!advertiser_info.p2p_advertiser_info.is_approved,
```

Comment:
looks like the error check and the comment above are out of date now then?

Reply:
Good catch! 👍

Reply:
Good catch! 👍